Dockerfile

```diff
@@ -92,13 +92,14 @@ RUN \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # install required python packages from file
-COPY pydeps.txt /tmp/pydeps.txt
-COPY pypkgs.txt /tmp/pypkgs.txt
+COPY deps_requirements.txt /tmp/deps_requirements.txt
```

Comment:
please rename to something like "install_requirements.txt". "deps_requirements" seems like doubly the same as "requirements" which come right below.

Reply:
`python_requirements.txt`? `install_requirements.txt` takes away the python context ;-)

Reply:
@kaspar030 good suggestions!